src/test/java/support/matchers/JsonObjectMatchers.java

```diff
@@ -0,0 +1,34 @@
+package support.matchers;
+
+import static org.hamcrest.Matchers.is;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+import com.jayway.jsonpath.matchers.JsonPathMatchers;
+
+import io.vertx.core.json.JsonObject;
+
+public final class JsonObjectMatchers {
+
```

Comment:
Personally, I wouldn't bother with the blank lines for between the class and the constructor